docs/docs/configuration/overview.md

```diff
@@ -81,6 +81,7 @@ An example [oauth2-proxy.cfg](https://github.com/oauth2-proxy/oauth2-proxy/blob/
 | `--auth-logging-format` | string | Template for authentication log lines | see [Logging Configuration](#logging-configuration) |
 | `--authenticated-emails-file` | string | authenticate against emails via file (one per line) | |
 | `--azure-tenant` | string | go to a tenant-specific or common (tenant-independent) endpoint. | `"common"` |
+| `--azure-use-workload-identity` | bool | whether to use Azure AD App federated credential to access Azure from within Kubernetes | false |
```

Comment:
Why specifically Kube?

Reply:
I would say that's the most common use case, but indeed it's not just kubernetes-specific. Will change it to something more generic.